src/apiCalls.js

```diff
@@ -0,0 +1,12 @@
+ function getMovies(id) {
+  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id || ''}`)
```

Comment:
I'm so stoked we were able to refactor to this!

Reply:
Yes this is cool 😎